src/actions/slack/utils.ts

```diff
@@ -62,6 +64,37 @@ const usableDMs = async (slack: WebClient): Promise<Channel[]> => {
     return users.map((user: any) => ({id: user.id, label: `@${user.name}`}))
 }
 
+export const setErrorResponse = (error: any) => {
```

Comment:
I added a function here which does something similar, except you define the HttpError and message to go to the function.  Can we re-use that?

https://github.com/looker-open-source/actions/blob/master/src/hub/action_response.ts#L22